plugins/catalog-backend-module-aws/api-report.md

```diff
@@ -19,11 +20,16 @@ import { UrlReader } from '@backstage/backend-common';
 // @public
 export type AWSCredentialFactory = (
   awsAccountId: string,
-) => Promise<Credentials>;
+) => Promise<AwsCredentialIdentity>;
```

Comment:
This is technically breaking, but they are very alike aren't they? I think we can let that one slide potentially.

Reply:
Correct.  `Credentials` appears to have behavior (although Backstage doesn't depend on it) whereas `AwsCredentialIdentity` does not.